SKILL.tsv

```diff
@@ -3975,60 +3975,60 @@ SKILL_20150414_003974	{memo X}Duration of [Install Distribution Table] Yogurt in
 SKILL_20150414_003975	$High Guard: Defense Increase
 SKILL_20150414_003976	{memo X}Defense of [High Guard] increases by 1 per attribute level.
 SKILL_20150414_003977	{memo X}Defense of [Swell Right Arm] increases by 1 per attribute level.
-SKILL_20150414_003978	Swell Brain: Enhance
+SKILL_20150414_003978	$Swell Brain: Enhance
 SKILL_20150414_003979	{memo X}Intelligence effect of [Swell Brain] increases per attribute level.
-SKILL_20150414_003980	Unbind: Enemy damage
+SKILL_20150414_003980	$Unbind: Enemy Damage
 SKILL_20150414_003981	{memo X}Enemies [Unbinded] receive damage. Attack increase by 10% of magic attack per attribute level.
-SKILL_20150414_003982	Summon Familar Spirit: Enhance
+SKILL_20150414_003982	$Summon Familar: Enhance
 SKILL_20150414_003983	{memo X}Damage of Spirit summoned by [Summon Familiar Spirit] increases per attribute level.
-SKILL_20150414_003984	Summon Familar Spirit: Splash
+SKILL_20150414_003984	$Summon Familar: Splash
 SKILL_20150414_003985	{memo X}When summoned familiar spirit explodes itself, there 1% chance per attribute level for it to damage nearby enemies.
-SKILL_20150414_003986	Sorcerer: SP Recovery
+SKILL_20150414_003986	$Sorcerer: SP Recovery
 SKILL_20150414_003987	{memo X}Character's SP recovery increase by 1 per attribute level.
 SKILL_20150414_003988	{memo X}Flesh Canon: Decompose
 SKILL_20150414_003989	{memo X}There is a chance of enemy falling under decompose status for 30 seconds when attacked with [Flesh Canon]. Chance increases by 5% per attribute level.
 SKILL_20150414_003990	{memo X}Flesh Hoop: Demoralize
 SKILL_20150414_003991	{memo X}Attak of enemies nearby the hoop after using [Flesh Hoop] decrease for 15 seconds. It turns back to normal when moving away from the hoop.
 SKILL_20150414_003992	Item Awakening: Quick
-SKILL_20150414_003993	Monsters created in dungeon with [Item Awakening] decreases by 1.
+SKILL_20150414_003993	$Decreases the amount of dungeon monsters with [Item Awakening] by 1.
 SKILL_20150414_003994	Item Awakening: Prompt Decision
-SKILL_20150414_003995	Wavescreated in dungeon with [Item Awakening] decreases by 1.
-SKILL_20150414_003996	Weapon Swap
+SKILL_20150414_003995	$Decreases the amount of created dungeon waves with [Item Awakening] by 1.
```

Comment:
I think "Decreases the amount of monster waves in a dungeon created with [Item Awakening] by 1" captures it better.